src/components/Game/Game.js

```diff
@@ -1,20 +1,49 @@
-import React, { useContext } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { Link } from "react-router-dom";
 
-import Button from "../Button";
 import "./Game.css";
-import { GameContext } from "../../contexts/gameContext";
+import Avatar from "../Avatar";
+import Button from "../Button";
+import PlayerCard from "../PlayerCard/PlayerCard";
 import ScoreInput from "../ScoreInput/ScoreInput";
+import Status from "../Status";
 import Swords from "../../images/swords.svg";
+import { GameContext } from "../../contexts/gameContext";
 import { Player } from "../../utils/player";
 
 const Game = props => {
   const game = useContext(GameContext);
   const [players, setPlayers] = game.usePlayers;
+  const [lowestPlayerLevel, setLowestPlayerLevel] = useState();
+  const [highestPlayerLevel, setHightestPlayerLevel] = useState();
+  const [isToggleSex, setIsToggleSex] = useState(false);
```

Comment:
This probably shouldn't be added as local state. You should re-use the `player.sex` value you have in context already and update that value on click!

Reply:
I refactored this. I am still not sure if it is the best way.